.github/workflows/R-CMD-check-build.yaml

```diff
@@ -0,0 +1,74 @@
+on:
+  workflow_call:
+    inputs:
+      RCLR_VERSION:
+        required: true
+        type: 'string'
+    secrets:
+      GITHUB_TOKEN:
+        required: true
+
+name: R-CMD-check-build
+
+jobs:
+  R-CMD-check-build:
+    runs-on: windows-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      R_KEEP_PKG_SOURCE: yes
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install PK-Sim
+        uses: Open-Systems-Pharmacology/Workflows/.github/actions/install-PKSIM
+
+      - name: Setup Pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install rClr
+        uses: Open-Systems-Pharmacology/Workflows/.github/actions/install-rClr
+        with:
+          RCLR_VERSION: ${{ inputs.RCLR_VERSION }}
+
+      - name: Install dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: |
+            rClr=local::./rClr_${{ env.RCLR_VERSION }}.zip
+            any::rcmdcheck
+            any::devtools
+            local::.
+            ospsuite.utils=github::Open-Systems-Pharmacology/OSPSuite.RUtils
+            tlf=github::Open-Systems-Pharmacology/TLF-Library
+            ospsuite=github::Open-Systems-Pharmacology/OSPSuite-R
+            ospsuite.parameteridentification=github::Open-Systems-Pharmacology/OSPSuite.ParameterIdentification
+          needs: check
+
+      - name: Check Package
+        uses: r-lib/actions/check-r-package@v2
+        with:
+          upload-snapshots: true
+          args: 'c("--no-manual", "--no-stop-on-test-error")'
+          build_args: 'c("--no-manual")'
+          error-on: 'c("error")'
+
+      - name: Build package
+        if: ${{ success() }}
+        run: |
+          output_dir <- file.path(normalizePath(Sys.getenv("RUNNER_TEMP"), winslash = "/"), "built_package")
+          dir.create(output_dir)
+          devtools::build(binary = TRUE, path = output_dir)
+        shell: Rscript {0}
+
+      - name: Upload built package
+        if: ${{ success() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: built_package
+          path: ${{ runner.temp }}/built_package/*.zip
```

Comment:
if the package should later run not only on Windows - I would create not only **.zip** but also **.tar.gz** version

Reply:
Sure ! But right now all the workflows are specifically made for windows.
We'll take care of that once the windows CI is 100% working and that rClr and all are Linux compatible.

Reply:
ok